bench/workgen/workgen.cxx

```diff
@@ -2379,8 +2409,35 @@ Workload::get_tables()
     return uris;
 }
 
+const std::vector<std::string>
+Workload::garbage_collection()
```

Comment:
I had thought the deletion of table structures would be part of the table deletion call and hence not needing a garbage_collection separate call. At this stage I would rather check-in this scheme and put more thought later on a different scheme:

My idea was for the remove call to return EBUSY if the table is in use. The python test would then call back again after sometime. If the table is free then, these structures would be removed.

I will think more over this as a separate ticket.